workflow/envs/bgzip.yaml

```diff
@@ -1,5 +1,6 @@
 channels:
-  - conda-forge
   - bioconda
+  - conda-forge
   - defaults
 dependencies:
+  - htslib=1.10
```

Comment:
I guess bgzip will use common, is htslib=1.10 and tabix=1.11 same?

Reply:
it seems, htslib/tabix/bgzip in `common:0.1.2` is `1.15.1` and not `1.11`